### DISCUSSION TITLE
Lima is significant slower than virt-manager?

### DISCUSSION THREAD

**Discussion by lattice0** [lima-vm/lima#1847] [Q&A]
I' m trying to move away from virt-manager and use lima because it's much easier to launch VMs and I don't really need the graphics now that I connect over VSCode tunnels.

Well, installing basic stuff like sudo apt install -y ... takes significantly more time than on a VM in virt-manager. Running on a arm64 i7 11th gen with 32gb RAM 8 cpus, and the guest has 10gb RAM, 8 cpus.

I'm running a ubuntu 23.04 host, creating the default ubuntu template without any modifications. Is there a reason for this? Could it be that it does not use virtiofs by default? This is the only thing I can think of.

**Comment by afbjorklund** [comment]
The virtfs (9p) and virtiofs should not be involved, package installation should use a local file system (probably over a virtio device, but anyway)

The most likely is that the default rootless networking for Lima, is slower than the default privileged networking used by Libvirt

Should only affect the download speed, though?

You might want to compare the other qemu flags too
